<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtDnaSequencesDialog</class>
 <widget class="QWidget" name="QtDnaSequencesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>522</width>
    <height>144</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLineEdit" name="edit_name_1">
     <property name="text">
      <string>1</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="edit_sequence_1">
     <property name="text">
      <string>ACGTA</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLineEdit" name="edit_name_2">
     <property name="text">
      <string>2</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="edit_sequence_2">
     <property name="text">
      <string>ACGTC</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLineEdit" name="edit_name_3">
     <property name="text">
      <string>3</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="edit_sequence_3">
     <property name="text">
      <string>ACGTG</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLineEdit" name="edit_name_4">
     <property name="text">
      <string>4</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="edit_sequence_4">
     <property name="text">
      <string>ACGTT</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
